clients/rust/tests/distribute_rewards.rs

```diff
@@ -72,6 +72,67 @@ async fn distribute_rewards() {
     );
 }
 
+#[tokio::test]
```

Comment:
Note: All of these "wrapped" tests were created by simply copy-pasting the success case and then configuring a wrapping reward rate on the system.